src/main/java/core/basesyntax/Model/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.Model;
+
+import core.basesyntax.Base.Machine;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working");
```

Comment:
The message in the `stopWork` method should also be more informative. It should clearly indicate that it is an `Excavator` that has stopped working. Refer to the checklist: 'Write informative messages in methods.'